src/components/Nav/AfterLogin/AfterLogin.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { BsBell } from 'react-icons/bs';
+import { AiFillHeart } from 'react-icons/ai';
+
+function AfterLogin() {
+  const [userData, setUserData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/loginData.json')
+      .then(res => res.json())
+      .then(data => setUserData(data[0]));
+  }, []);
+
+  return (
+    <AfterLoginBar>
+      <Slogan>코드 치다 지친 위코더여..술..GO? </Slogan>
+      <AfterContainer>
+        <BellButton>
+          <BsBell className="bell" />
+          <AiFillHeart className="heart" />
+        </BellButton>
+        <Link to="/mypage">
+          <UserProfile src={userData.image} alt="profile" />
+        </Link>
+        <Link to="/mypage">
+          <UserName>{userData.username} 위코더님</UserName>
+        </Link>
+        <LogoutButton>로그아웃</LogoutButton>
+      </AfterContainer>
+    </AfterLoginBar>
+  );
+}
+
+const AfterLoginBar = styled.div`
+  ${props => props.theme.flexSet('space-between')}
+  width: 100%;
+`;
+
+const Slogan = styled.h1`
+  color: ${props => props.theme.mint};
```

Comment:
```suggestion
  color: ${({theme}) => theme.mint};
```
다른 함수와 마찬가지로 매개변수 자리에서 구조분해할당 가능합니다!

Reply:
넵! 구조 분해 할당이 가능하다는 점 알려주셔서 감사합니다! 보다 간결하게 사용할 수 있어서 좋아요! 바로 적용 시켰습니다!!🙂